shared/metrics/src/main/java/io/pravega/shared/MetricsNames.java

```diff
@@ -152,12 +155,18 @@
     public static final String SLTS_SYSTEM_READ_BYTES = PREFIX + "segmentstore.storage.slts.system_read_bytes";     // Counter
     public static final String SLTS_SYSTEM_WRITE_BYTES = PREFIX + "segmentstore.storage.slts.system_write_bytes";   // Counter
     public static final String SLTS_CONCAT_BYTES = PREFIX + "segmentstore.storage.slts.concat_bytes";      // Counter
+
+    public static final String SLTS_READ_COUNT = PREFIX + "segmentstore.storage.slts.read_count";      // Counter
+    public static final String SLTS_WRITE_COUNT = PREFIX + "segmentstore.storage.slts.write_count";      // Counter
     public static final String SLTS_CREATE_COUNT = PREFIX + "segmentstore.storage.slts.create_count";      // Counter
     public static final String SLTS_DELETE_COUNT = PREFIX + "segmentstore.storage.slts.delete_count";      // Counter
     public static final String SLTS_CONCAT_COUNT = PREFIX + "segmentstore.storage.slts.concat_count";      // Counter
     public static final String SLTS_TRUNCATE_COUNT = PREFIX + "segmentstore.storage.slts.truncate_count";  // Counter
     public static final String SLTS_SYSTEM_TRUNCATE_COUNT = PREFIX + "segmentstore.storage.slts.system_truncate_count";  // Counter
 
+    public static final String SLTS_SYSTEM_READ_COUNT = PREFIX + "segmentstore.storage.slts.system_read_count";      // Counter
```

Comment:
segmentstore.storage.slts.system_ metrics are used to track writes to storage metadata segments.